### PR TITLE
Adding two traits for S144

### PR DESCRIPTION
**Pull request by david20000813**
Completing all traits for finite spaces (excluding semimetrizable, which will be completed once symmetrizable is added).

### CODE REVIEW

**Review by StevenClontz — COMMENTED** [review]
Would this be valid for the fixed point property? 

```
Let $f: X \to X$ be continuous, and note $x\leq y$ implies $f(x)\leq f(y)$. Then if neither $0$ nor $1$ are fixed points, then the range of $f$ must be $\{a\}$ or $\{b\}$, making either $a$ or $b$ a fixed point.
```

**Comment by david20000813** [comment]
@StevenClontz It’s basically the same proof, with far lower level of details. I don’t see any issue with it if people find this level of details is already sufficient.

**Review by StevenClontz — COMMENTED** [review]
Looks good to me. I'll defer to subscribe else to approve or as-is or second my suggestion to shorten the proof for the fixed point property.

**Comment by david20000813** [comment]
@danflapjax @JoeF131 In that case, would either of you be willing to chime in on this?

**Comment by prabau** [comment]
I think Steven's proof is fine, maybe adding one extra piece of information.  Namely, $x\le y$ iff $x\in\overline\\{y\\}$, i.e., corresponds to the specialization order.  And that's preserved by continuous function blah blah.

**Comment by david20000813** [comment]
@prabau Implemented.

File: spaces/S000144/properties/P000089.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+space: S000144
+property: P000089
+value: true
+---
+
+Note that in $X$, $x \leq y$ iff $x \in \overline{\{y\}}$, i.e., $\leq$ corresponds to the specialization preorder. Now, let $f: X\to X$ be continuous, then we have $x \leq y$ implies $f(x) \leq f(y)$. Hence, if neither $0$ nor $1$ are fixed points, the range of $f$ must be $\{a\}$ or $\{b\}$, making either $a$ or $b$ a fixed point.
```
Comment (prabau) [inline spaces/S000144/properties/P000089.md:7]:
```suggestion
Note that in $X$, $x \leq y$ iff $x \in \overline{\{y\}}$, i.e., $\leq$ corresponds to the specialization preorder, which is preserved by continuous functions. Let $f: X\to X$ be continuous. Then $x \leq y$ implies $f(x) \leq f(y)$. Hence, if neither $0$ nor $1$ are fixed points, the range of $f$ must be $\{a\}$ or $\{b\}$, making either $a$ or $b$ a fixed point.
```
minor tweak, seemed clearer to me

**Pull request merged**